src/plugins/GUITestBase/GUITestBase.pro

```diff
@@ -2,6 +2,13 @@ include (GUITestBase.pri)
 
 # Input
 HEADERS +=  src/GUITestBasePlugin.h \
+            src/harness/UGUITestBase.h \
```

Comment:
I'm really sorry, but "harness" doesn't look like the name with the obvious meaning)

Reply:
https://en.wikipedia.org/wiki/Test_harness
Test harness is a common name for the code that "supports testing" (not the test itself, but the infrastructure, helpers, etc...)